tests/c/simple_inline.newcg.c

```diff
@@ -0,0 +1,59 @@
+// ignore-if: test $YK_JIT_COMPILER != "yk" -o "$YKB_TRACER" = "swt"
+// Run-time:
+//   env-var: YKD_LOG_IR=-:jit-pre-opt
+//   env-var: YKD_SERIALISE_COMPILATION=1
+//   env-var: YKD_LOG_JITSTATE=-
+//   stderr:
+//     jitstate: start-tracing
+//     foo 7
+//     jitstate: stop-tracing
+//     --- Begin jit-pre-opt ---
+//     ...
+//     %{{result}}: i32 = Add %{{1}}, 3i32
+//     ...
+//     %{{2}}: i32 = Call @fprintf(%{{3}}, %{{4}}, %{{result}})
+//     ...
+//     --- End jit-pre-opt ---
+//     foo 6
+//     jitstate: enter-jit-code
+//     foo 5
+//     foo 4
+//     jitstate: deoptimise
+//     exit
+
+// Check that basic trace compilation works.
```

Comment:
This comment and the one below are inaccurate, right?

Reply:
[9115a7e](https://github.com/ykjit/yk/pull/1142/commits/9115a7e72d19eb383841ef20e55cfd367a369429)